content/news/XXXX-XX-XX-Midsummer-Mixxx.md

```diff
@@ -0,0 +1,65 @@
+title: Midsummer Mixxx
+authors: Evelynne Veys
+tags: summer, info, news, update, harmonic, key, stem
+comments: yes
+status: draft
+
+#### Let the Mixxx shine
+
+Summertime, partytime. In the middle of the festival season you are maybe playing a sunny vibes gig somewhere.
+Or you are relaxing on the beach or at the pool with a cocktail in the hand checking the long list of music you had to discover.
+Both ways will give you inspiration and energy.
+Who thinks Mixxx-devs are laying in the sun can't be more wrong, we're having a crowded github-festival ourselves.
+
+#### Let the Mixxx in
+
+A lot of new contributors found their way to our mapping parties on the discourse forum, our coding raves at github, 
+and the development zulip dancehall. You are all welcome and we'd like to thank you for all kind of input.
+With your feedback on the 2.5 beta version the coding team is on track for gold, the bugs and errors teams are already defeated.
+
+#### I'm going on a holiday and ..
+
+While your enjoying a well deserved vacation the Mixxx team is preparing new features.
+Not all new features are already for bundling in an official release but you can find everything about them in the partie venues. 
+
+The new visual aids for key and bpm will be astonishing, it needed quite some adaptations but the project is in pole position for gold.
```

Comment:
you are mixing the festival metaphor with the olympics... I would suggest using just one metaphor, probably the festival one since that's the way you started.  But I could also see a fun rewrite with all-olympics references!

Reply:
and even the past regional soccer cups :-)